### PR TITLE
change incorrect ids

### PR DESCRIPTION
**Pull request by goodviber**
Context: @lewisnix21  spotted an error in our BigQuery tables which is causing duplicate rows and preventing him from using the table. The issue is that the incentive ID '80464bd0-f509-4d54-b70a-add606b1844f'   is associated with 23/24 Applied Biology in one table and 2021 biology in another.
Fixes:
in the incentives table, ID 80464bd0-f509-4d54-b70a-add606b1844f should be replaced with 95eebd6c-69c5-4883-b3ff-c703b559bc4b (there should be two objects in the code with 95eebd6c-69c5-4883-b3ff-c703b559bc4b, one for classics, one for biology, both with the note 'classics and biology 2020-2021'
in the subjects table, ID 80464bd0-f509-4d54-b70a-add606b1844f should be replaced with 71730a26-b80b-440f-96ac-8c5814daa2c5  (there should be two objects in the code with 71730a26-b80b-440f-96ac-8c5814daa2c5, one for biology 23/24, one for applied biology 23/24

### CODE REVIEW

File: lib/dfe/reference_data/itt/incentives.rb [hunk 1]
```diff
@@ -451,20 +451,22 @@ module ITT
             scholarship: '17000',
             notes: 'geography 2020-2021'
           },
+          # rubocop:disable Lint/DuplicateHashKey
```
Comment (avinhurry) [inline lib/dfe/reference_data/itt/incentives.rb:454]: Why did you disable this cop?

File: lib/dfe/reference_data/itt/incentives.rb [hunk 2]
```diff
@@ -451,20 +451,22 @@ module ITT
...
           '95eebd6c-69c5-4883-b3ff-c703b559bc4b' => {
             subject_name: 'classics',
             itt_categories_id: '95547d49-2a52-4ae0-b4ea-744e0da4bfa2',
             academic_cycle: ['2021'],
             bursary_amount: '26000',
             notes: 'classics and biology 2020-2021'
           },
-          '80464bd0-f509-4d54-b70a-add606b1844f' => {
+          '95eebd6c-69c5-4883-b3ff-c703b559bc4b' => {
```
Comment (asatwal) [inline lib/dfe/reference_data/itt/incentives.rb:462]: This introduces a duplicate Hash key. It's same key as line 455, which shouldn't be allowed. We need resolve this duplicate. Also we need to keep the DuplicateHashKey cop for this reason.

File: lib/dfe/reference_data/itt/incentives.rb (hunk 1 above)
Comment (goodviber) [reply lib/dfe/reference_data/itt/incentives.rb:454]: see below, @lewisnix21 wanted 2 identical ids...

Comment (avinhurry) [reply lib/dfe/reference_data/itt/incentives.rb:454]: In Ruby, hash keys must be unique within the same hash, so defining the same key `95eebd6c-69c5-4883-b3ff-c703b559bc4b` twice will result in only the last occurrence being retained.

https://docs.ruby-lang.org/en/3.2/Hash.html

Comment (lewisnix21) [reply lib/dfe/reference_data/itt/incentives.rb:454]: @goodviber okay, after reviewing how other scenarios like this are handled in those two files (i.e. other groups of subjects with the same incentive notes), to be consistent and not introduce any duplicate hashing I propose we go with this:

1. Revert the change in `incentives.rb` so we use ID 80464bd0-f509-4d54-b70a-add606b1844f again
2. Where the ID 95eebd6c-69c5-4883-b3ff-c703b559bc4b appears for 2020-2021 Biology and 2020-2021 Applied Biology in `subjects.rb`, replace with the ID 80464bd0-f509-4d54-b70a-add606b1844f. Note the ID 95eebd6c-69c5-4883-b3ff-c703b559bc4b should remain for 2020-2021 Classics
3. Keep the current change to `subjects.rb` where you have replaced the ID for 2023-2024 Applied Biology from 80464bd0-f509-4d54-b70a-add606b1844f to 71730a26-b80b-440f-96ac-8c5814daa2c5

Comment (goodviber) [reply lib/dfe/reference_data/itt/incentives.rb]: Ok Kokey...

**Review by asatwal — APPROVED** [review]
LGTM 👍

**Pull request merged**